view/frontend/templates/js/replacejs.phtml

```diff
@@ -417,12 +417,14 @@ if (!$block->isEnabled()) return;
                 // sets the success order page redirect url from received data
                 // and calls the final Bolt defined callback
                 var onSuccess = function(data){
-
                     // show alert message on error
                     if (data.status !== 'success') {
                         if (data.message) {
                             console.log(data.message);
-                            alert(data.message);
+                            var boltModal = jQuery('#bolt-modal');
+
+                            boltModal.find('.bolt-modal-content').html('Error: ' + data.message);
```

Comment:
data.message here is not usually friendly. For now, let's use following message:
"Your payment was successful and we're now processing your order. If you don't receive order confirmation email in next 30 minutes, please contact us at support@bolt.com"